package.json

```diff
@@ -28,6 +31,7 @@
     "eslint-plugin-filenames": "^0.1.1",
     "eslint-plugin-mocha": "^0.2.2",
     "eslint-plugin-one-variable-per-var": "0.0.3",
+    "i18n-future": "^1.0.0",
```

Comment:
ah, sorry, it already is a dev dependency - sorry, misread that. Yes, only needed for testing.